cpp/src/groupby/sort/group_single_pass_reduction_util.cuh

```diff
@@ -40,14 +40,17 @@ struct reduce_functor {
   template <typename T>
   static constexpr bool is_supported()
   {
-    if (K == aggregation::SUM)
+    if (K == aggregation::SUM) {
```

Comment:
A switch statement seems more natural here, especially if the number of supported aggregations will grow.